src/MuiForms/EntitySelector.tsx

```diff
@@ -0,0 +1,74 @@
+import Autocomplete from '@mui/material/Autocomplete'
+import CircularProgress from '@mui/material/CircularProgress'
+import TextField from '@mui/material/TextField'
+import * as React from 'react'
+import { useListUsers } from '../../data/user'
+import { EntityKind } from '../../types/interfaces'
+
+interface Entity {
+  kind: string
+  id: string
+  data: any
+}
+
+export default function EntitySelector(props: any) {
+  const { users, isUsersLoading } = useListUsers()
```

Comment:
Instead of renaming below you could do `const { users, isUsersLoading: isLoading } = useListUsers()`